### DISCUSSION TITLE
Move "Clear Caches" into it's own toggle option in CP Access

### DISCUSSION THREAD

**Discussion by liamcrean** [ExpressionEngine/ExpressionEngine#2923] [Feature Requests]
Love template caching for the most part — apart from one detail; really do not like giving clients access to the Find and Replace utility — which includes database level replacement of text - which could do a LOT of damage in the wrong hands.

Clear Caches is very useful for site owners who want the power of template caching on site - but also wonder why their  minor changes arent visible... AFAIK The "Clear cache for new entries?" doesn't work for minor entry edits/updates/deletions etc? 

Moving "Clear Caches" into it's own toggle option in **> Roles: > CP Access** would be a great addition IMO - (or even just an option to hide F&R maybe?)

The other two utilities (_Search Reindex_ and _Manage Statistics_) do not contain potentially very destructive actions so not a problem.

**Comment by intoeetive** [comment]
https://github.com/ExpressionEngine/ExpressionEngine/pull/4777
